libraries/Microsoft.Bot.Builder.Skills/ChannelApiMiddleware.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Schema;
+
+namespace Microsoft.Bot.Builder.Skills
+{
+    /// <summary>
+    /// Handles InvokeActivity for ChannelAPI methods calls coming from SkillHostController.
```

Comment:
>SkillHostController [](start = 72, length = 19)

is this name valid?

Reply:
Changed it to skill adapter (no class name for now since this may change again in the future).